### PR TITLE
Add build support using go 1.3

### PR DESCRIPTION
**Pull request by ziransun**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/setup/setup.md [hunk 1]
```diff
@@ -16,7 +16,11 @@ KubeEdge is composed  of cloud and edge parts. It is built upon Kubernetes and p
 
 + [Creating kubernetes cluster with kubeadm on cloud side](<https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/>)
 
-+ **Go** The minimum required go version is 1.12. You can install this version by using [this website.](https://golang.org/dl/)
++ **Go** The minimum required go version is 1.12. You can install this version by using [this website.](https://golang.org/dl/). For go version 1.3, please run the following command first
```
Comment (rzr) [inline docs/setup/setup.md]: I assume you meant 1.13 ?

Relate-to: https://github.com/kubeedge/kubeedge/pull/1114

File: go13.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash 
+
+cp go13.mod go.mod
+cp go13.sum go.sum
```
Comment (kadisi) [inline go13.sh]: coping go.mod is not a good idea

**Comment by subpathdev** [comment]
If we want to make this change; we should move to go1.13 and not create a workaround and still live with go1.13

**Comment by subpathdev** [comment]
Can you change the used go version in all dockerfiles too?

**Comment by subpathdev** [comment]
@fisherxu agree we can go to go1.13

**Comment by rzr** [comment]
Is a release planned soon ? 

Anyway let me propose this change before updating to 1.13:
https://github.com/kubeedge/kubeedge/pull/1277

I will also test this go1.13 patch and can update this dockerfile (and others) accordingly.

**Comment by fisherxu** [comment]
Have supported go1.13 now #1349 :) Close the PR now, Thanks!

**Pull request closed without merging**